ons_alpha/urls.py

```diff
@@ -14,6 +14,22 @@
 from ons_alpha.search import views as search_views
 from ons_alpha.utils.cache import get_default_cache_control_decorator
 
+from wagtail.contrib.routable_page.models import RoutablePageMixin, route
+from ons_alpha.bulletins.views import latest_bulletin_view, previous_releases_view
+
+class BulletinSeriesPage(RoutablePageMixin, Page):
+    template = "templates/pages/bulletins/series_page.html"
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['BulletinPage']
+
+    @route(r'^latest/$')
+    def latest_bulletin(self, request):
+        return latest_bulletin_view(request, self)
+
+    @route(r'^previous-releases/$')
+    def previous_releases(self, request):
+        return previous_releases_view(request, self)
```

Comment:
both these methods do `page.get_latest_release()` / `page.get_previous_releases()`. You defined those methods under `BulletinPage`, but here you are passing an instance of `BulletinSeriesPage` which will explode